tests/test_testing.py

```diff
@@ -200,8 +200,8 @@ def test_empty_post_uses_default_boolean_value(self):
             data=None,
             content_type='application/json'
         )
-        self.assertEqual(response.status_code, 200, response.content)
-        self.assertEqual(response.data, {"flag": True})
```

Comment:
Third arg there is for an optional `msg` according to [docs](https://docs.python.org/2/library/unittest.html#unittest.TestCase.assertEqual). I think we can just drop it. I don't think we do that anywhere else.